nni/compression/pytorch/speedup/infer_mask.py

```diff
@@ -127,14 +127,17 @@ def random_init(self, start=0.1, end=8.0):
         # when the confidence is low. In the future, we will add the mask inference
         # rules for ReLU6 to break this range constraint.
         with torch.no_grad():
-            for tensor in self.dummy_input:
+            for index, tensor in enumerate(self.dummy_input):
                 if isinstance(tensor, torch.Tensor) and len(tensor.size()) > self.batch_dim\
                     and tensor.size(self.batch_dim) == self.batch_size:
                     # if the input tensor only has one dimension, which means
                     # it doesn't have the batch dimension, then we don't randomize
                     # this tensor, because our tensor scrambling is on the batch
                     # dimention. For example, if the tensor is a scalar(returned
                     # by the size operator), then we will skip this tensor
+                    if not tensor.is_contiguous():
```

Comment:
can `tensor` not a `torch.Tensor`?

Reply:
the tensor must be a torch.Tensor in this line